HPXMLtoOpenStudio/resources/constructions.rb

```diff
@@ -700,6 +858,9 @@ def self.apply_wood_frame_floor_ceiling(model, surfaces, constr_name, is_ceiling
       if not mat_int_finish_or_covering.nil?
         constr.add_layer(mat_int_finish_or_covering)
       end
+      if not mat_rb.nil?
+        constr.add_layer(mat_rb)
+      end
```

Comment:
This doesn't look like the right place. The order of construction layers here is insulation > framing > interior finish > radiant barrier. That means the radiant barrier is below the drywall, whereas it should be above the insulation. Comment applies to all ceiling methods.

Reply:
Now moved.